package.json

```diff
@@ -7,10 +7,12 @@
     "build": "NODE_ENV=production npm run build-components",
     "watch": "npm run watch-components",
     "build-components": "lerna run build --parallel",
+    "build-changed-components": "lerna run build --parallel --since",
     "watch-components": "lerna run watch --parallel",
     "setup-npm-links": "node scripts/setup-npm-link.js",
     "postinstall": "lerna bootstrap --hoist && cross-env NODE_ENV=production npm run build-components",
-    "release": "npm whoami && npx lerna clean && rm -rf node_modules && npm install && npm run test && lerna version && lerna publish from-package",
+    "prepublish": "cross-env NODE_ENV=production npm run build-changed-components",
```

Comment:
@markus-walther this will trigger on install as well: https://docs.npmjs.com/misc/scripts#description. You might want `prepublishOnly `

Reply:
Good point! Done